test/integration/verify/controls/aws_ec2_image.rb

```diff
@@ -0,0 +1,11 @@
+title 'Test a single AWS EC2 Image'
+
+control 'aws-ec2-image-1.0' do
+
+  impact 1.0
+  title 'Ensure AWS EC2 Image has current properties'
+
+  describe aws_ec2_image(image_id: '000000') do
```

Comment:
I think we can test a publicly available image that has most of the properties documented [here](docs/resources/aws_ec2_image.md). And, reference this control in the resource doc to give more example on how to use the resource.

Reply:
Ive added some testing for a public image and its properties.

Reply:
marking as resolved...